ui/repetition.reel/repetition.js

```diff
@@ -1881,57 +1893,45 @@ var Repetition = exports.Repetition = Component.specialize(/** @lends Repetition
     // isSelectionEnabled becoming false.
     _disableSelectionTracking: {
         value: function () {
-            if (window.Touch) {
-                this.element.removeEventListener("touchstart", this, true);
-            } else {
-                this.element.removeEventListener("mousedown", this, true);
-            }
+            this._pressComposer.removeEventListener("pressStart", this, false);
         }
     },
 
-    // ---
+    handlePressStart: {
+        value: function (pressEvent) {
+            var iteration = this._findIterationContainingElement(pressEvent.targetElement);
 
-    // Called by captureMousedown and captureTouchstart when a gesture begins:
-    /**
-     * @param pointerIdentifier an identifier that can be "mouse" or the
-     * "identifier" property of a "Touch" in a touch change event.
-     * @private
-     */
-    _observeSelectionPointer: {
-        value: function (pointerIdentifier) {
-            this._selectionPointer = pointerIdentifier;
-            this.eventManager.claimPointer(pointerIdentifier, this);
+            if (iteration) {
+                if (pressEvent.pointer === "mouse") {
+                    this._startX = pressEvent.event.clientX;
+                    this._startY = pressEvent.event.clientY;
 
-            var document = this.element.ownerDocument;
+                } else { // touch
+                    // The PressComposer is always getting the first touch.
+                    var touch = event.changedTouches[0];
 
-            if (window.Touch) {
-                // dispatches handleTouchend
-                document.addEventListener("touchend", this, false);
-                // dispatches handleTouchmove
-                document.addEventListener("touchmove", this, false);
-                // dispatches handleTouchcancel
-                document.addEventListener("touchcancel", this, false);
+                    this._startX = touch.clientX;
+                    this._startY = touch.clientY;
+                }
 
-            } else {
-                // dispatches handleMouseup
-                document.addEventListener("mouseup", this, false);
-                // dispatches handleMousemove
-                document.addEventListener("mousemove", this, false);
+                this.__pressComposer.addEventListener("press", this, false);
+                this.__pressComposer.addEventListener("pressCancel", this, false);
+
+                this.element.addEventListener("touchmove", this, false);
+                document.addEventListener("scroll", this, true);
```

Comment:
We already talked about it and we came up with this solution to disable an iteration when we are scrolling. Listening to mousemove is not needed here because the PressComposer will check if we are still on the same target when we release the mouse. Plus, I believe we can't scroll with mousemove, right ?

It's not managed by the PressComposer because it can be tricky in some cases, for example with a big button we can still move on it between a TouchStart and a TouchEnd.